Editor/Scripts/MeshSyncServerInspector.cs

```diff
@@ -85,6 +86,38 @@ public void DrawServerSettings(MeshSyncServer t)
         }
     }
 
+    static void DrawDCCToolInfo(MeshSyncServer server)
+    {
+        if (server != null)
+        {
+            GUILayout.BeginHorizontal();
+
+            EditorGUI.BeginChangeCheck();
+
+            server.m_DCCAsset = EditorGUILayout.ObjectField("DCC asset file:", server.m_DCCAsset, typeof(UnityEngine.Object), true);
+            if (server.m_DCCAsset != null)
+            {
+                if (GUILayout.Button("Open"))
```

Comment:
This should also be renamed to "Live Edit"

Reply:
Fixed

Reply:
Done